client/src/views/Feed/SinglePost.vue

```diff
@@ -0,0 +1,305 @@
+      <div class="qCard-wrapper" ref="qCard">
+      <q-card id="feed-modal-video">
+        <q-card-section>
+          <template>
+          <div class="video-container" @click="play">
+            <figure class="image is-32x32">
+              <avatar :img="post.avatarUrl" :size="'30px'" />
+            </figure>
+            <video ref="modalVideo" @click.once="incrementViewsCounter(post)">
+                <q-resize-observer @resize="onResize" />
+                <source :src="post.videoUrl" type="video/mp4" />
+                <source :src="post.videoUrl" type="video/ogg" />
+                <source :src="post.videoUrl" type="video/webm" />
+            </video>
+          </div>
+          <div class="content-wrapper">
+            <div class="heart-and-comments">
+              <div class="heart">
+                <i class="far fa-heart fa-lg"
+                :class="{'fas': isLiked}"
+                @click="updateLikes(post)">
+                </i>
+                <p class="likes">{{ this.post.likesCount }}</p>
+              </div>
+              <div class="comments-icon" @click="closePopup(true)">
+                <i class="far fa-comment-alt fa-md"></i>
+                <p class="likes">{{ this.commentsCount(this.post.postId) }}</p>
+              </div>
+            </div>
+            <div class="content">
+              <p
+                ref="caption"
+                :class="[post.caption.length > 40 ? 'caption-in-modal' : 'caption-in-modal']"
+              >
+                <router-link :to="{ path: `/profile/${post.nickname}` }">
+                  <span>{{ post.nickname }}</span>
+                </router-link>
+                {{ post.caption }}
+              </p>
+            </div>
+          </div>
+          </template>
+        </q-card-section>
+      </q-card>
+      </div>
+    </q-dialog>
+  </div>
+</template>
+
+<script>
+import PageComments from '../Comments/PageComments'
+import Avatar from '@/components/Avatar.vue'
+import { updateLikes, incrementViewsCounter } from '../../services/posts'
+import { mapGetters, mapActions } from 'vuex'
+
+export default {
+  name: 'OnePost',
+  props: {
+    post: {
+      type: Object,
+      default: null
+    }
+  },
+  data () {
+    return {
+      isModelVisible: false,
+      dialog: false,
+      likesInfo: {
+        show: false
+      },
+      isLiked: false,
+      showNotAuth: false
+    }
+  },
+  computed: {
+    ...mapGetters({ myProfile: 'profile/myProfile' }),
+    ...mapGetters({ comments: 'comments/myComments' }),
+    ...mapGetters({ commentsCount: 'comments/commentsCount' })
+  },
+
+  methods: {
+    ...mapActions({
+      dislikePost: 'posts/dislikePost',
+      likePost: 'posts/likePost'
+    }),
+    isPostLiked (post) {
+      const likesArr = this.post.likes
+      if (likesArr.includes(localStorage.getItem('profileId'))) {
+        this.isLiked = true
+      } else {
+        this.isLiked = false
```

Comment:
this.isLiked = likesArr.includes(localStorage.getItem('profileId')) ???

Reply:
It is current user id. (for now)
To know **is post liked** by a current user or **no**